test/EventListener/MessengerListenerTest.php

```diff
@@ -14,7 +15,7 @@
 
 class MessengerListenerTest extends BaseTestCase
 {
-    /** @var \Prophecy\Prophecy\ObjectProphecy|FlushableClientInterface */
+    /** @var \Prophecy\Prophecy\ObjectProphecy|ClientInterface */
```

Comment:
With the newest versions, this should work:
```suggestion
    /** @var \Prophecy\Prophecy\ObjectProphecy<ClientInterface> */
```

Reply:
I tried and it doesn't work, it gives a bunch of errors the first of which is:

`PHPDoc tag @var for property Sentry\SentryBundle\Test\EventListener\MessengerListenerTest::$client contains generic type Prophecy\Prophecy\ObjectProphecy<Sentry\ClientInterface> but class Prophecy\Prophecy\ObjectProphecy is not generic.`

Reply:
Nevermind, I updated the `jangregor/phpstan-prophecy` package and now it works!